src/terminal/adapter/conGetSet.hpp

```diff
@@ -89,7 +92,8 @@ namespace Microsoft::Console::VirtualTerminal
         virtual BOOL PrivateEraseAll() = 0;
         virtual BOOL SetCursorStyle(const CursorType cursorType) = 0;
         virtual BOOL SetCursorColor(const COLORREF cursorColor) = 0;
-        virtual BOOL PrivateGetConsoleScreenBufferAttributes(_Out_ WORD* const pwAttributes) = 0;
+        virtual BOOL PrivateGetConsoleScreenBufferLegacyAttributes(_Out_ WORD* const pwAttributes) = 0;
+        virtual BOOL PrivateGetConsoleScreenBufferAttributes(_Out_ TextAttribute* const pAttributes) = 0;

```

Comment:
I've got good news! In the long while we've let this PR sit (sorry), @j4james added `PrivateSetTextAttributes` and `PrivateGetTextAttributes` which may act as a serviceable replacement for `PrivateSetAttributes` and `PrivateGetConsoleScreenBufferAttributes`.

Reply:
Sweet!